test/Telegram.Bot.Tests.Integ/Stickers/StickersTests.cs

```diff
@@ -2,18 +2,24 @@
 using Telegram.Bot.Tests.Integ.Common;
 using Telegram.Bot.Types;
 using Xunit;
+using System.Linq;
+using Telegram.Bot.Types.Enums;
 
 namespace Telegram.Bot.Tests.Integ.Stickers
 {
     [Collection(CommonConstants.TestCollections.Stickers)]
     [TestCaseOrderer(CommonConstants.TestCaseOrderer, CommonConstants.AssemblyName)]
-    public class StickersTests
+    public class StickersTests : IClassFixture<StickersTestsFixture>
     {
+        private readonly StickersTestsFixture _classFixture;
+
         private readonly TestsFixture _fixture;
 
-        public StickersTests(TestsFixture assemblyFixture)
+        public StickersTests(StickersTestsFixture classFixture)
         {
-            _fixture = assemblyFixture;
+            _classFixture = classFixture;
+            _fixture = _classFixture.TestsFixture;
+            _classFixture.ChatId = ConfigurationProvider.TestConfigurations.ChannelChatId;
```

Comment:
Channel ID should be set on TestsFixture. Also, no need to send to channel, supergroup is fine.

Reply:
Tests should be run in channels only when we expect a different behavior.